content/en/docs/configuration/venafi.md

```diff
@@ -32,6 +32,13 @@ resources, read the [Namespaces](../../concepts/issuer/#namespaces) section.
 
 ### Creating a Venafi Cloud Issuer
 
+⚠ From cert-manager `v1.3` you will need to update your Venafi Cloud configuration to use `OutagePREDICT` instead of `DevOpsACCELERATE`.
+With this update, the zone format changes from a UUID to a string of the form `<Application Name>\<Issuing Template Alias>`.
+Please read [cert-manager 1.2 to 1.3 upgrade notes][] and [Venafi Cloud Prerequisites][] for further information.
+
+[cert-manager 1.2 to 1.3 upgrade notes]: ../../installation/upgrading/upgrading-1.2-1.3/
+[Venafi Cloud Prerequisites]: https://github.com/Venafi/vcert/blob/v4.13.1/README-CLI-CLOUD.md#prerequisites
```

Comment:
```suggestion
{{% pageinfo color="warning" %}}

⚠ From cert-manager `v1.3` onwards, you will need to update your Venafi Cloud
configuration to use `OutagePREDICT` instead of `DevOpsACCELERATE`. With this
update, the zone format changes from a UUID to a string of the form:

```text
    <Application Name>\<Issuing Template Alias>
```

Please read [cert-manager 1.2 to 1.3 upgrade notes][] and [Venafi Cloud
Prerequisites][] for further information.

[cert-manager 1.2 to 1.3 upgrade notes]: ../../installation/upgrading/upgrading-1.2-1.3/
[venafi cloud prerequisites]: https://github.com/Venafi/vcert/blob/v4.13.1/README-CLI-CLOUD.md#prerequisites

{{% /pageinfo %}}
```

It adds a nice red square to outline the warning:

![Screenshot 2021-04-07 at 17 22 47](https://user-images.githubusercontent.com/2195781/113892092-fa61d880-97c5-11eb-9255-1e69241036d2.png)